cloudify/utils.py

```diff
@@ -360,7 +360,7 @@ def exception_to_error_cause(exception, tb):
     etype = type(exception)
     traceback.print_exception(etype, exception, tb, file=error)
     return {
-        'message': str(exception),
+        'message': exception,
```

Comment:
```suggestion
        'message': u'{0}'.format(exception),
```
for a 2+3 compatible turning the exception into unicode
(as opposed to the previous str()-ing, which wasa turning it into unicode or bytes)

Reply:
Updating to use that approach